api/spec/requests/api/v1/search_results_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "swagger_helper"
+
+RSpec.describe "Search Results", type: :request do
+
+  context "can search a project", :elasticsearch do
```

Comment:
Did I do this right? It looks like this test is still trying to run and failing.

Reply:
The tag was correct. The issue was with your spec — you were reindexing models outside of a before block and that code was being executed whether or not the examples were included. Generally, doing things outside of let and before/after blocks in specs is an antipattern and you shouldn't do it.

Reply:
See line 9 below for the correct way to do what you were trying to do.